integrations/sms/config.json

```diff
@@ -0,0 +1,47 @@
+{
+    "display_name": "SMS",
+    "description": "Send Service alerts via SMS.",
+    "integration_type": "event_output",
+    "supported_event_types": [],
+    "required_fields": [],
+    "max_send_retries": 5,
+    "polling_enabled": false,
+    "test_connection_enabled": true,
+    "parameters": [
+        {
+            "value": "to_phone",
+            "label": "TO (phone number)",
+            "type": "string",
+            "required": true,
+            "help_text": "A phone number that will receive the SMS"
```

Comment:
Any phone syntax requirements? + dashes, etc.?

Reply:
Added format example to the help text.